pod/perlvar.pod

```diff
@@ -1008,6 +1009,8 @@ find uses of these problematic match variables in your code.
 
 =over 8
 
+=item Digit Vars
```

Comment:
I understand you're trying to make more linkable headings with this and Branch Reset, and I don't have a particular problem with doing it this way, just noting it's unorthodox for these documents.

Reply:
(The perlvar addition in particular may require me to special case it in Perldoc Browser to avoid considering "Digit Vars" a new variable name, but that's my problem.)

Reply:
Personal preference would be to spell out "Variables" in headings.

Reply:
Removed.